tests/gradients/test_finite_difference.py

```diff
@@ -407,12 +410,12 @@ def expval(self, observable, **kwargs):
 
         dev = DeviceSupportingSpecialObservable(wires=1, shots=None)
 
-        @qml.qnode(dev, diff_method="parameter-shift")
+        @qml.qnode(dev, diff_method="finite-diff")
         def qnode(x):
             qml.RY(x, wires=0)
             return qml.expval(SpecialObservable(wires=0))
 
-        @qml.qnode(dev, diff_method="parameter-shift")
+        @qml.qnode(dev, diff_method="finite-diff")

```

Comment:
To check up to good numerical precision, we should choose the same method here as above.